grammar.js

```diff
@@ -570,6 +574,7 @@ module.exports = grammar({
     ),
 
     statement: $ => seq(
+      optional($._explain_statement),
```

Comment:
```suggestion
      optional(seq(
        $.keyword_explain,
        optional($.keyword_analyze),
        optional($.keyword_verbose),
      )),
```
`explain` won't be reused and we can cut out the precedence